backend/api/Controllers/RobotController.cs

```diff
@@ -573,6 +573,69 @@ public async Task<ActionResult> ResumeMission([FromRoute] string robotId)
         return NoContent();
     }
 
+
+    /// <summary>
+    /// Post new armposition (Battery pose, Transport pose, Lookout pose) for the robot with id 'robotId'
+    /// </summary>
+    /// <remarks>
+    /// <para> This query moves the arm to a given position for a given robot </para>
+    /// </remarks>
+    [HttpPut]
+    [Authorize(Roles = Role.User)]
+    [Route("{robotId}/SetArmPosition/{armPosition}")]
+    [ProducesResponseType(typeof(Mission), StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    public async Task<ActionResult> SetArmPosition(
+        [FromRoute] string robotId,
+        [FromRoute] string armPosition
+    )
+    {
+        var robot = await _robotService.ReadById(robotId);
+        if (robot == null)
+        {
+            _logger.LogWarning("Could not find robot with id={id}", robotId);
+            return NotFound("Robot not found");
```

Comment:
```suggestion
           var errorMessage = $"Could not find robot with id {robotId}";
            _logger.LogWarning(errorMessage);
            return NotFound(errorMessage);
```

Reply:
It might be C# doesn't like logging the errorMessage variable directly.

Reply:
done